fuzzbench/jobs.py

```diff
@@ -11,25 +11,30 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
```

Comment:
I don't think it's a good idea to have all jobs/tasks in one module. They will do totally different things and this file will become a mess.

Reply:
> lszekeres yesterday Member
Let's just have a jobs.py instead of fake_jobs and build_jobs and use the build_image function currently in build_jobs

I agree with you. But as @lszekeres suggested above, we can separate the logic later and then keep it simple now.

Reply:
once we implement all these functions (build/run/measure), it will definitely make sense to separate them

Reply:
Fine for now